include/openssl/ec.h

```diff
@@ -708,10 +708,11 @@ size_t EC_POINT_point2buf(const EC_GROUP *group, const EC_POINT *point,
                           unsigned char **pbuf, BN_CTX *ctx);
 
 /* other interfaces to point2oct/oct2point: */
-BIGNUM *EC_POINT_point2bn(const EC_GROUP *, const EC_POINT *,
-                          point_conversion_form_t form, BIGNUM *, BN_CTX *);
-EC_POINT *EC_POINT_bn2point(const EC_GROUP *, const BIGNUM *,
-                            EC_POINT *, BN_CTX *);
+DEPRECATEDIN_3_0(BIGNUM *EC_POINT_point2bn(const EC_GROUP *, const EC_POINT *,
+                                           point_conversion_form_t form,
+                                           BIGNUM *, BN_CTX *))
+DEPRECATEDIN_3_0(EC_POINT *EC_POINT_bn2point(const EC_GROUP *, const BIGNUM *,
+                                             EC_POINT *, BN_CTX *))
```

Comment:
Please use the newer deprecation method with `OSSL_DEPRECATEDIN_3_0`, see https://github.com/openssl/openssl/blob/master/doc/internal/man7/deprecation.pod#header-files

Reply:
Not sure I quite understand why this is better..
We currently have > 650 instances in headers of DEPRECATEDIN_3_0

Reply:
I have changed it for this PR.